### ISSUE TITLE
Import itevents.io calendar data to get additional events for digest

### ISSUE THREAD

**Issue by webknjaz** [itkpi/itkpimail#17]
@rrader @zlot777 any comments on this idea?
Perhaps, we could crawl http://dou.ua/calendar as well..

**Comment by rrader** [comment]
We have to parse several sources and put results to the Suggested Events with "sugested_by" field filled by specific source (this field usually filled with username who suggested the event).

Possibly crawler should have separate account and be independent application using ITKPIMail API to suggest events.

**Comment by webknjaz** [comment]
Yeah, I like the idea of having separate accounts for different sources. Let's call it `crawer-bot`.
I think we could use `scrapy` to crawl dou.ua and maybe some other event sources.
